deploy/entrypoint.sh

```diff
@@ -2,11 +2,32 @@
 
 set -e
 
-# install manfiests and wait to be ready
+# stop the tiller if running
+function stop_tiller {
+    supervisord ctl shutdown
+}
+trap stop_tiller EXIT
+
+# install manfiests and wait to be ready (e.g. crds)
 kubectl apply --validate=false -f /manifests/
 kubectl wait --for condition=established --timeout=${TIMEOUT:-60s} -f /manifests/
 
+# create mysql-operator orchestrator topology secret
+orc_secret_name=stack-mysql-operator-topology-credentials
+if ! kubectl -n ${NAMESPACE:-presslabs-system} get secret $orc_secret_name; then
+    cat <<EOF | kubectl create -f-
```

Comment:
So that we don't save the secret content in it's annotation.

```suggestion
    cat <<EOF | kubectl create --save-config=false -f-
```